packages/core/graph/src/Graph.js

```diff
@@ -289,11 +335,17 @@ export default class Graph<TNode, TEdgeType: number = 1> {
     ) {
       return this.dfsFast(enter, startNodeId);
```

Comment:
Would be nice to have a comment here to explain why there are two different dfs implementations

Reply:
Added JSDoc comments